components/divider/stories/divider.stories.js

```diff
@@ -44,18 +46,83 @@ export default {
 		rootClass: "spectrum-Divider",
 		size: "m",
 		vertical: false,
-		customStyles: {
-			"min-inline-size": "200px",
-		},
 	},
 	parameters: {
 		componentVersion: version,
 	},
 };
 
+/**
+ * The default size for divider is medium.
+ */
 export const Default = DividerGroup.bind({});
 Default.args = {};
 
+// ********* DOCS ONLY ********* //
+export const Sizing = (args, context) => Sizes({
+	Template,
+	withHeading: false,
+	withBorder: false,
+	...args,
+}, context);
+Sizing.tags = ["!dev"];
+Sizing.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * When a vertical divider is used inside of a flex container, use `align-self: stretch; block-size: auto;` on the divider.
+ */
+export const VerticalGroup = (args, context) => Sizes({
```

Comment:
```suggestion
export const VerticalSizing = (args, context) => Sizes({
```

^ Propose aligning the story's name with the story above since they're both pushing through the same Sizes function.

Reply:
Sure thing: https://github.com/adobe/spectrum-css/pull/2833/commits/0be603d2bb21b21308a838698a6380c92560581b